index copy.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Vacation planner</title>
+</head>
+
+<body style="background-color: rgb(210, 227, 195);">
+    <h1>Personal vacation planner</h1>
+    <a href="index.html">Home |</a>
+    <a href="bio.html">Bio |</a>
+    <a href="schedule.html">Schedule</a>
+    <p><i>Please choose the time period, then click on the <b>Submit for approval</b> button</i></p>
+    <h2>Time period</h2>
+    <p>Start date</p><input type="date">
+    <p>End date</p><input type="date">
+    <p></p><input type="button" , <body style="background-color: rgb(104, 210, 136);" value="Submit for approval">
```

Comment:
Aici ai un paragraf care nu contine nimic si acel "body" care nu ar trebui sa fie acolo.
De asemenea, ai o virgula in interiorul inputului care nu este ok acolo.